src/main/java/com/training/services/ga/validate/Header.java

```diff
@@ -0,0 +1,9 @@
+package com.training.services.ga.validate;
```

Comment:
Why is the loyalty pojos under - com.training.services.ga.validate?
This should be under - com.training.services.ga.loyalty